docs/develop/javascript-objects.md

```diff
@@ -142,6 +142,10 @@ async function main(context: Excel.RequestContext) {
 
 ```
 
+## Using external JavaScript libraries
```

Comment:
Nit: Arguably, the section title is misleading, given the content of the section. Consider renaming it to something like "Use of external JavaScript libraries is not supported"

Reply:
Good call. Updated.